scripts/stac/imagery/import_metadata.py

```diff
@@ -0,0 +1,19 @@
+from enum import Enum
+from typing import Optional
+
+import xmltodict
+from linz_logger import get_log
+
+
+class MetadataType(str, Enum):
+    EARTHSCANNER = "earthscanner"
+
+
+def get_cloud_percent(xml_input: str) -> Optional[int]:
+    try:
+        input_metadata = xmltodict.parse(xml_input)
+        cloud_percent: int = int(input_metadata["MetaData"]["ProductInfo"]["CloudPercent"])
```

Comment:
do we know if this is always a `int` ? could it be a `float` ?

Reply:
From the examples I've looked at, they were always integer. For example `JL1KF01C_PMSR3_20230308055539_200139323_102_0020_001_L3C_PSH_meta.xml` has a value of `14`. I could parse it as a float, but wonder if want to have a value like `14.00` in the stac metadata for this instance?